### DISCUSSION TITLE
Extension stopped working after 10k blocks!

### DISCUSSION THREAD

**Discussion by biencriado** [kheina-com/Blue-Blocker#326] [General]
Very happy with the extension, I set it up some time ago with block rate of 1/5 seconds and let it work in the background. 

About a month ago, I was approaching the 10k accounts blocked, and then an error message appeared saying blocking permissions have been revoked and the extension stopped working on my device and I was logged off. 

I had the extension in another device as well, and it stopped working in that one as well. Is there a workaround for this issue?

**Comment by kheina** [comment]
do you happen to have a screenshot of the error message you received? I think there is a maximum number of users you can block nowadays, so I don't think this is an issue with blue blocker. try switching to the mute option

**Comment by ZakilStone** [comment]
Same problem here, the extension capped at 10k but I know for a fact you can go over that in twitter. 
![j2](https://github.com/user-attachments/assets/2b4f19d0-d22a-470d-a53f-75f2ead90ea1)
![j](https://github.com/user-attachments/assets/cde1fb0f-3327-4c99-b649-b4b0a0f82181)

**Comment by cooljeanius** [comment]
Are you sure this isn't just due to the issue with AMO refusing to approve our updates to the Firefox version? See #349
